src/main/resources/fabric.mod.json

```
@@ -2,15 +2,21 @@
   "schemaVersion": 1,
   "id": "team_reborn_energy",
   "version": "${version}",
-  "name": "Energy",
-  "description": "An energy API",
+  "name": "Energy by Team Reborn",
```

Thats not the name of it.
```suggestion
  "name": "Energy",
```

---

alright, seemed a little generic